ansible_base/lib/utils/views/permissions.py

```diff
@@ -1,5 +1,24 @@
+from django.conf import settings
 from rest_framework.permissions import SAFE_METHODS, BasePermission
 
+oauth2_provider_installed = "ansible_base.oauth2_provider" in getattr(settings, "INSTALLED_APPS", [])
```

Comment:
There is be an `ansible_base.lib.utils.settings get_setting` function which will search in both the services settings as well as the actual settings. It might be "safer" to use that here (for now).